### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -1,45 +1,124 @@
...
+        genre = self.get_object(pk)
+        genre.delete()
+
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class ActorList(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    generics.GenericAPIView,
```
Comment (Yaroslav-Stetsko) [inline cinema/views.py:76]: You can use 
generics.ListCreateAPIView

File: cinema/views.py [hunk 2]
```diff
@@ -1,45 +1,124 @@
...
+        return self.list(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        return self.create(request, *args, **kwargs)
+
+
+class ActorDetail(
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.DestroyModelMixin,
```
Comment (Yaroslav-Stetsko) [inline cinema/views.py:91]: You can use
generics.RetrieveUpdateDestroyAPIView

**Review by Yaroslav-Stetsko — APPROVED** [review]
Great job

File: cinema/serializers.py [hunk 3]
```diff
@@ -1,6 +1,63 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(required=True, max_length=255)
```
Comment (Y-Havryliv) [inline cinema/serializers.py:8]: https://www.django-rest-framework.org/api-guide/fields/#required
fix everywhere

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: cinema/serializers.py (hunk 3 above)
Comment (HStakhiv) [reply cinema/serializers.py]: Less code – less trouble

**Review by Y-Havryliv — APPROVED** [review]
Nice!

**Pull request closed without merging**